spec/support/v5.0/sorbet_test_cases.rb

```diff
@@ -312,7 +312,11 @@
 T.assert_type!(HogwartsAcceptanceMailer.notify(wizard), ActionMailer::MessageDelivery)
 
 # Jobs
-AwardHousePointHourglasses.perform_later(student: wizard, point: 10)
+T.assert_type!(
+  AwardHousePointHourglasses.perform_later(student: wizard, point: 10),
+  AwardHousePointHourglasses
+)
```

Comment:
Can you update the test case in the generator file instead of the generated file? Once you updated it in the generator, please run `rake update_spec:sorbet_test_cases`

https://github.com/chanzuckerberg/sorbet-rails/blob/master/spec/generators/sorbet_test_cases.rb